xstream/sinks/mqtt_sink.go

```diff
@@ -95,15 +110,24 @@ func (ms *MQTTSink) Configure(ps map[string]interface{}) error {
 		}
 	}
 
+	retained := false
+	if pk, ok := ps["retained"]; ok {
```

Comment:
Update the document `docs/en_US/rules/sinks/mqtt.md` and `docs/zh_CN/rules/sinks/mqtt.md`.